api/src/main/java/javax/application/event/EventHandler.java

```diff
@@ -30,6 +30,22 @@
 @Retention(value = RetentionPolicy.RUNTIME)
 @Target(value = {ElementType.METHOD, ElementType.ANNOTATION_TYPE})
 public @interface EventHandler {
+
+    /**
+     * Default value for a low priority.
+     */
+    int LOW_PRIORITY = -100;
+
+    /**
+     * Default priority
+     */
+    int DEFAULT_PRIORITY = 0;
+
+    /**
+     * Default value for a high priority.
+     */
+    int HIGH_PRIORITY = 100;
```

Comment:
why restrict these values to `-100` and `100`? I'd leave open to interpretation for each implementor to decide the number range, same as it is with `Comparable` and implementations of `compareTo(O)`

Reply:
I was thinking just as a hint for them. But yes, maybe we should start just with the default value. Will change it